src/app/handlers/acknowledge.handler.ts

```diff
@@ -0,0 +1,26 @@
+import { MessageReaction, User } from 'discord.js';
+import { IContainer, IHandler } from '../../common/types';
+
+export class AcknowledgeHandler implements IHandler {
+  constructor(public container: IContainer) {}
+
+  public async execute(reaction: MessageReaction, user: User): Promise<void> {
+    //Get Member from user ID
+    const member = this.container.guildService.get().members.find((m) => m.id === user.id);
```

Comment:
Try https://discord.js.org/#/docs/main/stable/class/GuildMemberManager?scrollTo=fetch

```js
members.fetch(user.id)
```
should be faster than O(n) search

Reply:
guild.members is a `Collection` not a `GuildMemberManager`, so it doesn't have a fetch method

Reply:
if it's a `Collection` then the `.get()` should take the user id (type `Snowflake`)